Vagrantfile

```diff
@@ -31,6 +31,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   end
 
   config.vm.provider "virtualbox" do |vb, override|
+    config.vm.synced_folder ".", "/srv/zulip", type: "rsync", rsync__args: ["--verbose", "--archive", "--delete", "-z"]
```

Comment:
It's the second answer in the SO topic you posted earlier (http://stackoverflow.com/questions/24200333/symbolic-links-and-synced-folders-in-vagrant). Per the Vagrant docs, "Vagrant can use rsync as a mechanism to sync a folder to the guest machine... The rsync synced folder type accepts the following options:

rsync__args (array of strings) - A list of arguments to supply to rsync. By default this is ["--verbose", "--archive", "--delete", "-z", "--copy-links"]."

--copy-links is causing the linking problem, so we configure the synced_folder without it